src/views/Pools/components/LockedPool/Common/BalanceField.tsx

```diff
@@ -36,19 +36,22 @@ const BalanceField: React.FC<PropsType> = ({
   const [percent, setPercent] = useState(0)
   const { userNotEnoughCake, notEnoughErrorMessage } = useUserEnoughCakeValidator(lockedAmount, stakingTokenBalance)
 
+  useEffect(() => {
+    const amount = new BigNumber(lockedAmount)
+    if (amount.gt(0)) {
+      const convertedInput = amount.multipliedBy(BIG_TEN.pow(stakingDecimals))
```

Comment:
BIG_TEN.pow(stakingDecimals) can be extracted to a state by using initial state as function. Therefore we will not calculate it each render

Reply:
Ignore my comment above, the correct approach should be a create a state with initalstate as function  and a useEffect that sets that state when stakingDecimals change.